frontend/src/pages/acceleratorProfiles/screens/list/AcceleratorProfilesTableRow.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Text, TextContent, TextVariants, Timestamp, Truncate } from '@patternfly/react-core';
+import { ActionsColumn, Td, Tr } from '@patternfly/react-table';
+import { AcceleratorKind } from '~/k8sTypes';
+import AcceleratorProfileEnableToggle from '~/pages/acceleratorProfiles/components/AcceleratorProfileEnableToggle';
+
+type AcceleratorProfilesTableRow = {
+  accelerator: AcceleratorKind;
+};
+
+const AcceleratorProfilesTableRow: React.FC<AcceleratorProfilesTableRow> = ({ accelerator }) => (
+  <Tr>
+    <Td dataLabel="Name">
+      <TextContent>
+        <Text>{accelerator.spec.displayName}</Text>
+        {accelerator.spec.description && (
+          <Text component={TextVariants.small}>
+            <Truncate content={accelerator.spec.description} />
+          </Text>
+        )}
+      </TextContent>
+    </Td>
+    <Td dataLabel="Identifier">{accelerator.spec.identifier}</Td>
+    <Td dataLabel="Enable">
+      <AcceleratorProfileEnableToggle
+        enabled={accelerator.spec.enabled}
+        name={accelerator.spec.displayName}
+      />
+    </Td>
+    <Td dataLabel="Last modified">
+      {accelerator.metadata.annotations?.['opendatahub.io/modified-date'] ? (
+        <Timestamp
+          date={new Date(accelerator.metadata.annotations['opendatahub.io/modified-date'])}
```

Comment:
invalid dates render as _now_. We should render invalid dates as `--`

Reply:
How the invalid date looks like? Because if the data does not exists it is already displaying `--`

Reply:
It renders at the date.now()

Reply:
So the issue is with the Timestamp component. see this description of default date: https://v4-archive.patternfly.org/v4/components/timestamp#default

TLDR it default to now if its not defined (or invalid)

to test if a date string is invalid you can use `Date. parse()` which will return `NaN` if it is invalid. Then you can use the `isNaN()` function to test that

Reply:
Yeah looks like the mocks use a relative time anyway. so just use that function instead of timestmap.